aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/UpdateHandler.java

```diff
@@ -122,16 +123,18 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     }
                     return progress;
                 }, CallbackContext::isReadReplicaPromoted, CallbackContext::setReadReplicaPromoted))
-                .then(progress -> Commons.execOnce(progress, () ->
-                                versioned(proxy, rdsProxyClient, progress, null, ImmutableMap.of(
+                .then(progress -> Commons.execOnce(progress, () -> {
+                                    progress.getCallbackContext().timestampOnce(RESOURCE_UPDATED_AT, Instant.now());
```

Comment:
before that point we could have sent up to 2 modify calls. Might be worth to set the variable earlier